install_bootstrap

```diff
@@ -323,6 +326,7 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         azul_s3_bucket='azul_s3_bucket'
         ask_question "What is the name of the AWS S3 bucket to store BDBags?" "$AZUL_S3_BUCKET" "Azul S3 Bucket" $azul_s3_bucket
 
+	azul_presigned_url_expiration='azul_presigned_url_expiration'
```

Comment:
This should be `azul_presigned_url_expiration=3600`, or whatever the default value is supposed to be.

Reply:
fixed